fvm/evm/genesis-mainnet.json

```
@@ -0,0 +1,37 @@
+{
+  "config": {
+    "chainId": 747,
+    "homesteadBlock": 0,
+    "daoForkBlock": 0,
+    "daoForkSupport": false,
+    "eip150Block": 0,
+    "eip155Block": 0,
+    "eip158Block": 0,
+    "byzantiumBlock": 0,
+    "constantinopleBlock": 0,
+    "petersburgBlock": 0,
+    "istanbulBlock": 0,
+    "muirGlacierBlock": 0,
+    "berlinBlock": 0,
+    "londonBlock": 0,
+    "arrowGlacierBlock": 0,
+    "grayGlacierBlock": 0,
+    "mergeNetsplitBlock": 0,
+    "shanghaiTime": 0,
+    "cancunTime": 0,
+    "terminalTotalDifficulty": 0,
+    "ethash": {}
+  },
+  "nonce": "0x0",
+  "timestamp": "0x66d3ae80",
+  "extraData": "0x",
+  "gasLimit": "0x7270E00",
+  "difficulty": "0x20000",
+  "mixHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+  "coinbase": "0x0000000000000000000000000000000000000000",
```

Mainnet is using `0x0000000000000000000000030000000000000000` as `coinbase`, but for testnet, the coinbase was `0x0000000000000000000000000000000000000000`. We changed the coinbase address to `0x000...3000...` for testnet through an HCU, but for mainnet, it's started with 3. 

https://github.com/onflow/flow-go/blob/master/fvm/evm/types/address.go#L41
```suggestion
  "coinbase": "0x0000000000000000000000030000000000000000",
```

---

Good catch :+1: Though there were no transactions involved during the genesis block, hence no fees were transferred to some coinbase. But anyway, let's go with the suggestion, seems legit.